src/Knp/Menu/Twig/MenuRuntimeExtension.php

```diff
@@ -7,6 +7,9 @@
 use Knp\Menu\Util\MenuManipulator;
 use Twig\Extension\RuntimeExtensionInterface;
 
+/**
+ * @final since 3.8.0
```

Comment:
The MenuRuntimeExtension itself cannot become `@internal` as its instantiation has to be a public API to be usable by projects registering the runtime into Twig (KnpMenuBundle in the common case, but that's still a separate package), but all the methods can be `@internal` as they are only meant to be referenced by the MenuExtension as implementation of the (public) Twig callables.